tests/integration-tests-topologies/src/main/resources/cube-definitions/3-node-all-version-unstarted.yaml

```diff
@@ -0,0 +1,64 @@
+#/**
+# * Copyright 2007 The Apache Software Foundation
+# *
+# * Licensed to the Apache Software Foundation (ASF) under one
+# * or more contributor license agreements.  See the NOTICE file
+# * distributed with this work for additional information
+# * regarding copyright ownership.  The ASF licenses this file
+# * to you under the Apache License, Version 2.0 (the
+# * "License"); you may not use this file except in compliance
+# * with the License.  You may obtain a copy of the License at
+# *
+# *     http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# */
+networks:
+  testnetwork*:
+    driver: bridge
+
+zookeeper*:
+  image: jplock/zookeeper
```

Comment:
nit: any reason why no use an "official" image? shall we include the version here, otherwise, it will fetch the latest image, no?

Reply:
ah, last time I looked there wasn't an official one, and jplock was the most popular, so it imprinted on my brain. Will use official.